src/pretix/control/permissions.py

```diff
@@ -90,3 +90,33 @@ class OrganizerPermissionRequiredMixin:
     def as_view(cls, **initkwargs):
         view = super(OrganizerPermissionRequiredMixin, cls).as_view(**initkwargs)
         return organizer_permission_required(cls.permission)(view)
+
+
+def administrator_permission_required(permission):
```

Comment:
If we don't use it, we can drop the permission parameter ;) In that case, we'll need to think about ``@administrator_permission_required`` vs ``@administrator_permission_required()``. Ideally, both should work, which is not easy to achieve, but IIRC ``functools`` provides a utility to support this.